libs/langchain/langchain/vectorstores/aerospike.py

```diff
@@ -0,0 +1,3 @@
+from langchain_community.vectorstores.aerospike import Aerospike
```

Comment:
this sshould NOT be re-exported inside Langchain, should be community only

Reply:
Ok, sorry about that. I tried to replicate what I saw